earth_enterprise/rpms/pre-uninstall_fusion.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: requires xmllint from libxml2-utils
+
+set +x
+
+# get script directory
+# config values
+ASSET_ROOT=""
+GEFUSIONUSER_NAME=""
+GROUPNAME=""
+
+# directory locations
+BININSTALLROOTDIR="/etc/init.d"
+
+# script arguments
+HAS_EARTH_SERVER=false
+
+#-----------------------------------------------------------------
+# Main Functions
+#-----------------------------------------------------------------
+main_preuninstall()
+{
+    service gefusion stop
+
+    load_systemrc_config
+
+    GROUP_EXISTS=$(getent group $GROUPNAME)
```

Comment:
These variables seem to be unused now.  You could remove them.

Reply:
ok,